src/marqo/tensor_search/backend.py

```diff
@@ -149,5 +149,8 @@ def get_cluster_indices(config: Config):
     """Gets the name of all indices"""
     res = HttpRequests(config).get(path="_aliases")
     indices = set(res.keys())
-    relevant_indices = indices - constants.INDEX_NAMES_TO_IGNORE
+    indices_no_protected_prefixes = {index_name for index_name in indices
```

Comment:
nit (i.e. not in this PR): this logic probably should be in a separate function.